docs/cli.md

```
@@ -0,0 +1,906 @@
+| --- | --- | --- | --- |
+| `--tail <n>` | number | 200 | Tail last N log lines |
+
+### hack log-pipe
+
+Usage: `hack log-pipe [options]`
+
+Options:
+
+| Flag | Type | Default | Description |
+| --- | --- | --- | --- |
+| `--format <auto|docker-compose|plain>` | string | `auto` | How to parse incoming lines from stdin |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)